Zenodo fixes (#162)
echemdb/electrochemistry-data: README.md

# Electrochemistry-data

<p align="center">
  <img src="https://img.shields.io/badge/Code_License-GPL_3.0_or_later-blue.svg" alt="Code License: GPL 3.0 or later">
  <a href="https://creativecommons.org/licenses/by/4.0/"><img src="https://img.shields.io/badge/Data_License-CC--BY--4.0-blue.svg" alt="Data License: CC-BY-4.0"></a>
  <a href="https://opendatacommons.org/licenses/by/"><img src="https://img.shields.io/badge/Data_License-ODC--By--1.0-blue.svg" alt="Data License: ODC-By-1.0"></a>

>   <a href="https://doi.org/10.5281/zenodo.20723429"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.20723429.svg" alt="DOI"></a>

Reviewer: Is this the permanent DOI or the release specific one?

Author: permanent, and willpoint to the lates version.

</p>

This repository hosts a dataset for interfacial electrochemistry data,

> currently focusing on cyclic voltammograms of single crystal electrodes

Reviewer: ```suggestion
focusing on cyclic voltammograms of well-defined single crystal electrodes
```

recorded in aqueous and non-aqueous electrolytes. It consists of published
data that is either retraced from source PDFs or provided by the authors, and
can be browsed on [echemdb.org/cv](https://www.echemdb.org/cv).

> ![alt text](doc/figures/echemdb_overview.png)

Reviewer: ```suggestion
![Cyclic voltammograms of select transition metal electrodes.](doc/figures/echemdb_overview.png)
```

> The data is provided as frictionless [`unitpackages`](https://echemdb.github.io/unitpackage/)

Reviewer: ```suggestion
The data is provided as frictionless-based [`unitpackages`](https://echemdb.github.io/unitpackage/)
```

and originates from two kinds of input:

* **Digitized data** retraced from figures in publications, stored as SVG and
  YAML files and converted with [`svgdigitizer`](https://echemdb.github.io/svgdigitizer/).

> * **Directly submitted data** provided by authors as raw CSV files together with

Reviewer: ```suggestion
* **Directly submitted data** provided by authors as CSV files together with
```
